packages/dds/tree/src/test/shared-tree/sharedTree.bench.ts

```diff
@@ -431,4 +432,69 @@ describe("SharedTree benchmarks", () => {
 			}
 		}
 	});
+
+	// In this context "op bunch" refers to a group of ops for the same DDS that are sent by a peer in a single message.
+	describe("rebasing over op bunch", () => {
+		// The number of commits in a bunch for a given run of this test suite.
+		const bunchSizes = isInPerformanceTestingMode ? [1, 10, 100] : [10];
+		// Number of local commits to rebase over the inbound bunch
+		const localBranchSizes = isInPerformanceTestingMode ? [10, 100] : [2];
+		// If op bunching is used, then the time taken to rebase the local branch over the bunch should be proportional to `bunchSize + localBranchSize` instead of `bunchSize * localBranchSize`.
+		// For the following timings:
+		// +----------------------+-------------+--------------+
+		// |                      | bunchSize:1 | bunchSize:10 |
+		// +----------------------+-------------+--------------+
+		// | localBranchSize: 10  | t1          | t2           |
+		// | localBranchSize: 100 | t3          | t4           |
+		// +----------------------+-------------+--------------+
+		// Then, if op bunching is used, then t4 should be roughly equal to t1 + (t2-t1) + (t3-t1)
```

Comment:
@yann-achard-MS Do we want to keep this part of the comment? I am not sure I follow this.

Reply:
Also, I added a bunch size of 100, so I should either update this table in the comment or remove 100.

Reply:
Removed the 100 bunch size.